README.md

```diff
@@ -57,7 +57,7 @@ Auto-populate your redux store with this configuration data instead of setting d
 
 ### `style.css`
 
-Add a stylesheet that gets appended at the end of your entry stylesheet. This allows you to override styles for specific deployments and use [custom CSS properties](http://cssnext.io/features/#custom-properties-var).
+Add a stylesheet that gets `@import`ed at the beginning of your entry stylesheet. This allows you to override styles for specific deployments and use [custom CSS properties](http://cssnext.io/features/#custom-properties-var). Useful for configuration specific images and colors.
```

Comment:
@trevorgerhardt I think you forgot to include `style.css` in the list of config files above: https://github.com/conveyal/mastarm/pull/146/commits/d1bdc4ff274ae6eca97264e0689669cd781f69f7#diff-04c6e90faac2675aa89e2176d2eec7d8R32